src/crystal/system/unix/syscall/arm-linux.cr

```diff
@@ -0,0 +1,459 @@
+    CLOCK_ADJTIME                = 372
+    SYNCFS                       = 373
+    SENDMMSG                     = 374
+    SETNS                        = 375
+    PROCESS_VM_READV             = 376
+    PROCESS_VM_WRITEV            = 377
+    KCMP                         = 378
+    FINIT_MODULE                 = 379
+    SCHED_SETATTR                = 380
+    SCHED_GETATTR                = 381
+    RENAMEAT2                    = 382
+    SECCOMP                      = 383
+    GETRANDOM                    = 384
+    MEMFD_CREATE                 = 385
+    BPF                          = 386
+    EXECVEAT                     = 387
+    USERFAULTFD                  = 388
+    MEMBARRIER                   = 389
+    MLOCK2                       = 390
+    COPY_FILE_RANGE              = 391
+    PREADV2                      = 392
+    PWRITEV2                     = 393
+    PKEY_MPROTECT                = 394
+    PKEY_ALLOC                   = 395
+    PKEY_FREE                    = 396
+    STATX                        = 397
+    RSEQ                         = 398
+    IO_PGETEVENTS                = 399
+    MIGRATE_PAGES                = 400
+    KEXEC_FILE_LOAD              = 401
+    CLOCK_GETTIME64              = 403
+    CLOCK_SETTIME64              = 404
+    CLOCK_ADJTIME64              = 405
+    CLOCK_GETRES_TIME64          = 406
+    CLOCK_NANOSLEEP_TIME64       = 407
+    TIMER_GETTIME64              = 408
+    TIMER_SETTIME64              = 409
+    TIMERFD_GETTIME64            = 410
+    TIMERFD_SETTIME64            = 411
+    UTIMENSAT_TIME64             = 412
+    PSELECT6_TIME64              = 413
+    PPOLL_TIME64                 = 414
+    IO_PGETEVENTS_TIME64         = 416
+    RECVMMSG_TIME64              = 417
+    MQ_TIMEDSEND_TIME64          = 418
+    MQ_TIMEDRECEIVE_TIME64       = 419
+    SEMTIMEDOP_TIME64            = 420
+    RT_SIGTIMEDWAIT_TIME64       = 421
+    FUTEX_TIME64                 = 422
+    SCHED_RR_GET_INTERVAL_TIME64 = 423
+    PIDFD_SEND_SIGNAL            = 424
+    IO_URING_SETUP               = 425
+    IO_URING_ENTER               = 426
+    IO_URING_REGISTER            = 427
+    OPEN_TREE                    = 428
+    MOVE_MOUNT                   = 429
+    FSOPEN                       = 430
+    FSCONFIG                     = 431
+    FSMOUNT                      = 432
+    FSPICK                       = 433
+    PIDFD_OPEN                   = 434
+    CLONE3                       = 435
+    CLOSE_RANGE                  = 436
+    OPENAT2                      = 437
+    PIDFD_GETFD                  = 438
+    FACCESSAT2                   = 439
+    PROCESS_MADVISE              = 440
+    EPOLL_PWAIT2                 = 441
+    MOUNT_SETATTR                = 442
+    QUOTACTL_PATH                = 443
+    LANDLOCK_CREATE_RULESET      = 444
+    LANDLOCK_ADD_RULE            = 445
+    LANDLOCK_RESTRICT_SELF       = 446
+  end
+
+  # TODO: Use correct assembly for ARM
```

Comment:
According to "Architecture calling conventions" in [this man page](https://man7.org/linux/man-pages/man2/syscall.2.html):

Arch/ABI | Instruction | System Call # | Ret Val | arg1 | arg2 | arg3 | arg4 | arg5 | arg6
---|---|---|---|---|---|---|---|---|---
arm/EABI  |  `swi 0x0` | r7 | r0 | r0  | r1  | r2 |  r3  | r4 |  r5
arm64  |    `svc #0`    |   w8   |   x0 | x0  |  x1  |  x2  |  x3   | x4  |  x5